modules/apps/static/portal-dependency-manager-component-executor-factory/portal-dependency-manager-component-executor-factory/src/main/java/com/liferay/portal/dependency/manager/component/executor/factory/internal/DependencyManagerSyncImpl.java

```diff
@@ -171,15 +171,11 @@ private void _registerDependencyManagerSyncGatekeeperCallable() {
 
 				_componentContext.enableComponent(componentName);
 
-				while (true) {
-					if (countDownLatch.await(1, TimeUnit.MINUTES)) {
-						_componentContext.disableComponent(componentName);
+				countDownLatch.await();
 
-						bundleContext.removeServiceListener(serviceListener);
+				_componentContext.disableComponent(componentName);
 
-						break;
-					}
-				}
+				bundleContext.removeServiceListener(serviceListener);
```

Comment:
@yuhai Please do not have a separate commit to modify what you just added, modify the original commit directly.